server/svix-server/src/worker.rs

```diff
@@ -153,12 +154,12 @@ async fn process_failure_cache(
                 &FailureCacheValue {
                     first_failure_at: now,
                 },
-                expiration_period,
+                disable_in * 2,
```

Comment:
Add a comment to say that this means that we "forgive" failures after 2 * disable_ins.